<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">alternate-background-color: rgb(20, 235, 235);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_1_Home">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2_Home">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_Home">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min1_Home">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Home">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 0, 255);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Away">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(26, 0, 255);</string>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1_Away">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2_Away">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_Away">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min1_Away">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetAll">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_puluhmenit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_menit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_puluhdetik">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_detik">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>16</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_startTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pauseTime">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
    <property name="shortcut">
     <string>Space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetTime">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set15min">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>15min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set10min">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set5min">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set3min">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set1min">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tambah1menit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tambah1detik">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+1det</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Ronde">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 21, 21);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_ronde">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tambahronde">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kurangironde">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_foulHome">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(219, 8, 8);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kurangifoulhome">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tambahfoulhome">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_foulAway">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 16, 16);</string>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tambahfoulaway">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kurangifoulaway">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>85</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>24 detik</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_puluhdetik_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_detik_24">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tet">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>tet</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Home">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Away">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <zorder>pushButton_1_Home</zorder>
   <zorder>pushButton_2_Home</zorder>
   <zorder>pushButton_3_Home</zorder>
   <zorder>pushButton_min1_Home</zorder>
   <zorder>lcdNumber_Home</zorder>
   <zorder>lcdNumber_Away</zorder>
   <zorder>pushButton_1_Away</zorder>
   <zorder>pushButton_2_Away</zorder>
   <zorder>pushButton_3_Away</zorder>
   <zorder>pushButton_min1_Away</zorder>
   <zorder>pushButton_resetAll</zorder>
   <zorder>label_puluhmenit</zorder>
   <zorder>label_menit</zorder>
   <zorder>label_puluhdetik</zorder>
   <zorder>label_detik</zorder>
   <zorder>label_7</zorder>
   <zorder>pushButton_startTime</zorder>
   <zorder>pushButton_pauseTime</zorder>
   <zorder>pushButton_resetTime</zorder>
   <zorder>pushButton_set15min</zorder>
   <zorder>pushButton_set10min</zorder>
   <zorder>pushButton_set5min</zorder>
   <zorder>pushButton_set3min</zorder>
   <zorder>pushButton_set1min</zorder>
   <zorder>pushButton_tambah1menit</zorder>
   <zorder>pushButton_tambah1detik</zorder>
   <zorder>lcdNumber_Ronde</zorder>
   <zorder>label_ronde</zorder>
   <zorder>pushButton_tambahronde</zorder>
   <zorder>pushButton_kurangironde</zorder>
   <zorder>lcdNumber_foulHome</zorder>
   <zorder>pushButton_kurangifoulhome</zorder>
   <zorder>pushButton_tambahfoulhome</zorder>
   <zorder>lcdNumber_foulAway</zorder>
   <zorder>pushButton_tambahfoulaway</zorder>
   <zorder>pushButton_kurangifoulaway</zorder>
   <zorder>label_puluhdetik_24</zorder>
   <zorder>label_detik_24</zorder>
   <zorder>pushButton_tet</zorder>
   <zorder>lineEdit_Home</zorder>
   <zorder>lineEdit_Away</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBantuan"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionTime_Score"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBantuan">
   <property name="text">
    <string>Panduan (F1)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTime_Score">
   <property name="text">
    <string>S and T</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
